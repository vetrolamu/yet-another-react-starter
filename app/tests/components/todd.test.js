import React from 'react';
import Header from '../../components/todd/todd.jsx';
import renderer from 'react-test-renderer';

it('renders correctly', () => {
    const tree = renderer.create(
        <Header title="Hello" />
    ).toJSON();
    expect(tree).toMatchSnapshot();
});
