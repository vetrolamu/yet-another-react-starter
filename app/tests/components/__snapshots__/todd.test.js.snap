exports[`test renders correctly 1`] = `
<div
  className="todd">
  <h1
    title="Todd">
    Todd
  </h1>
  <div>
    Hi, I am Todd. Look at 
    <a
      onClick={[Function bound handleClick]}
      style={Object {}}>
      my todos
    </a>
  </div>
  <img
    src="FILE_MOCK"
    width="200" />
  <br />
</div>
`;
